web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -4,23 +4,29 @@
 - Status: Accepted
 - Discussion: [#47444](https://github.com/pandas-dev/pandas/pull/47444)
 - Author: [Marc Garcia](https://github.com/datapythonista)
```

Comment:
It probably makes sense to list this PR in the `Discussion` header next to the original one, and having you as a co-author here. I think that will help people find the discussions and know who to contact in case of something related to the PR.